crates/spice_cloud/src/lib.rs

```diff
@@ -45,6 +61,110 @@ impl SpiceExtension {
     pub fn new(manifest: ExtensionManifest) -> Self {
         SpiceExtension { manifest }
     }
+
+    fn control_plane_enabled(&self) -> bool {
+        self.manifest
+            .params
+            .get("control_plane_enabled")
+            .unwrap_or(&"false".to_string())
+            == "true"
```

Comment:
This should probably be a case-insensitive compare

Reply:
fixed, thanks